### PR TITLE
Pulsedive unexpected response bug fix

### PR DESCRIPTION
**Pull request by AnastasiiaRozlyvan**
An error occurred when a list was received instead of a string. Screenshot 2020-04-08 at 10 04 18
Screenshot 2020-04-08 at 09 55 12
And when POST request was sent to observe/observable with empty value, result was : "message": "{:error \"{:errors [{:message (not (instance? java.lang.String a-clojure.lang.PersistentArrayMap))}]}\"} [:invalid-server-response]".

### CODE REVIEW

File: api/enrich.py [hunk 1]
```diff
@@ -228,6 +228,24 @@ def get_relationship(source_ref, target_ref, relationship_type):
             }
 
 
+def get_related_ips(observable, ips):
+    relations = []
+    if ips:
+        if type(ips) != list:
```
Comment (GeorgiyII) [inline api/enrich.py:234]: isinstance()?

File: api/enrich.py [hunk 2]
```diff
@@ -244,13 +262,10 @@ def extract_sightings(output, unique_indicator_ids, sightings_relationship):
...
-    related = output['properties'].get('dns', {}).get('A')
-    relations = {
-                'origin': 'Pulsedive Enrichment Module',
-                'related': {'type': 'ip', 'value': related},
-                'relation': 'Resolved_To',
-                'source': observable,
-            }
+    related_ips = get_related_ips(
+        observable,
+        output['properties'].get('dns', {}).get('A')
```
Comment (GeorgiyII) [inline api/enrich.py:267]: get('A', [])
and you and you shouldn't check if ips

File: api/enrich.py (hunk 1 above)
Comment (AnastasiiaRozlyvan) [reply api/enrich.py]: Thanks ! This is a good idea.  Done.

File: api/enrich.py (hunk 2 above)
Comment (AnastasiiaRozlyvan) [reply api/enrich.py]: Done.

**Pull request merged**